source/Mlos.Python/OptimizerEvaluationTools.ipynb

```diff
@@ -0,0 +1,394 @@
+{
+ "cells": [
```

Comment:
I highly recommend viewing this vile fully rendered. I left the output there for ease of reviewing.